python/spglib/__init__.py

```diff
@@ -33,6 +33,14 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+
+from ._version import (  # noqa: F401
+    __version__,
+    __version_tuple__,
+    version,
+    version_tuple,
```

Comment:
Do we need `version` and `version_tuple`? I thought `__vesion__` and `__version_tuple__` would be sufficient.

Reply:
Probably the latter two are sufficient, I'll remove them later